docs/tutorial_GPU.ipynb

```diff
@@ -0,0 +1,5769 @@
+{
```

Comment:
<p>you could consider putting all of these into a dictionary (e.g. <code>options = dict(...)</code>)  and then passing it in the next cell as <code>Filter(..., **options)</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocean-eddy-cpt/gcm-filters/pull/69/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>